pkg/knuu/instance_helper.go

```diff
@@ -314,3 +314,45 @@ func (i *Instance) addFileToBuilder(src string, dest string, chown string) error
 	}
 	return nil
 }
+
+// deployIngress deploys the ingress for the instance
+func (i *Instance) deployIngress() error {
+	ingress := i.ingress
+
+	annotations := map[string]string{}
+
+	if ingress.BackendProtocol != "" {
+		annotations["nginx.ingress.kubernetes.io/backend-protocol"] = ingress.BackendProtocol
+	}
+
+	if ingress.CertManagerEnabled {
+		//annotations["kubernetes.io/tls-acme"] = "true"
```

Comment:
please explain why we need this comment out

Reply:
This was a leftover from testing. We have this annotation in our kustomize manifests files, but the value is deprecated.
I removed this comment in: https://github.com/celestiaorg/knuu/pull/151/commits/2035aa15f62223ca002f5311c14c2493858854bc